### PR TITLE
Fe bottom buttons

### PR DESCRIPTION
**Pull request by gracu-koduje**
Please review my code

### CODE REVIEW

File: js/scripts.js [hunk 1]
```diff
@@ -1,17 +1,62 @@
...
-    currentDeg = 0;
+    currentDeg = 0,
+    circles = $(".icon-bottom i");
+
 
 $(".next").on("click", { d: "n"}, rotate);
 $(".prev").on("click", { d: "p"}, rotate);
 
+function colorCircle() {
+    $("[data-dot-id='1']").toggleClass("test", currentDeg == 0 || currentDeg == 360);
```
Comment (matwjablonski) [inline js/scripts.js]: i think you can write in as a function with params

File: js/scripts.js [hunk 2]
```diff
@@ -1,17 +1,62 @@
...
+    $("[data-dot-id='4']").toggleClass("test", currentDeg == -180 || currentDeg == 180);
+    $("[data-dot-id='5']").toggleClass("test", currentDeg == -240 || currentDeg == 120);
+    $("[data-dot-id='6']").toggleClass("test", currentDeg == -300 || currentDeg == 60);
+    console.log(carousel);
+}
+colorCircle();
+
 function rotate(e) {
-    if (e.data.d=="n") {
+    if (e.data.d == "n" && currentDeg >= -300) {
```
Comment (matwjablonski) [inline js/scripts.js]: better is to compare by `===`

File: js/scripts.js [hunk 3]
```diff
@@ -1,17 +1,62 @@
...
         currentDeg = currentDeg - 60;
     }
-    if (e.data.d=="p") {
+    if (e.data.d == "n" && currentDeg < -300) {
```
Comment (matwjablonski) [inline js/scripts.js]: `===`

File: js/scripts.js [hunk 4]
```diff
@@ -1,17 +1,62 @@
...
+        currentDeg = 0;
+    }
+    if (e.data.d == "p" && currentDeg <= 300) {
```
Comment (matwjablonski) [inline js/scripts.js]: `===`

File: js/scripts.js [hunk 5]
```diff
@@ -1,17 +1,62 @@
...
         currentDeg = currentDeg + 60;
     }
+    if (e.data.d == "p" && currentDeg > 300) {
```
Comment (matwjablonski) [inline js/scripts.js]: `===`

File: js/scripts.js [hunk 6]
```diff
@@ -1,17 +1,62 @@
...
 
     carousel.css({"transform": "rotateY(" + currentDeg + "deg)"});
-
+    colorCircle();
 }
+
+circles.click(function () {
+    var currentCircle = $(this).attr("data-dot-id");
+
+    if (currentCircle == 1) {
```
Comment (matwjablonski) [inline js/scripts.js]: you can use switch here

File: js/scripts.js (hunk 1 above)
Comment (gracu-koduje) [inline js/scripts.js]: Done!

File: js/scripts.js (hunk 2 above)
Comment (gracu-koduje) [inline js/scripts.js]: Done!

File: js/scripts.js (hunk 3 above)
Comment (gracu-koduje) [inline js/scripts.js]: Done!

File: js/scripts.js (hunk 4 above)
Comment (gracu-koduje) [inline js/scripts.js]: Done!

File: js/scripts.js (hunk 5 above)
Comment (gracu-koduje) [inline js/scripts.js]: Done!

File: js/scripts.js (hunk 6 above)
Comment (gracu-koduje) [inline js/scripts.js]: Done!

**Pull request merged**